Yoast/YoastSEO.js
# Add language list for morphology

## Summary

This PR can be summarized in the following changelog entry:

* [not user-facing] Adds a helper that returns a list of languages that have word forms support

## Relevant technical choices:

* Export this function in the helpers' index. Which is loaded in the root index

## Test instructions

This PR can be tested by following these steps:

* Run the tests
* When linked to the plugin you can run the following in the console `yoast.analysis.helpers.getFormsLanguages()` and it should return an array with just `en` inside. It being there tests it got exported. English there matches the currently only available morphology language.

Fixes #2156
